### PR TITLE
Show more skills

### PR DESCRIPTION
**Pull request by LubosGangster**
sorry it took me so long, couldn't done it sooner than today, cause i was busy

### CODE REVIEW

**Review by TammyTee — CHANGES_REQUESTED** [review]
Hi @LubosGangster!

Thanks for working on this. I wonder if we might be able to take a much simpler approach to achieve this by using [AlpineJS](https://alpinejs.dev/start-here) (already included in the codebase) to show/hide extra skills right on the frontend. 

Would you mind looking into that and let me know what you think? Thanks!

**Comment by TammyTee** [comment]
Hi @LubosGangster - is there anything I can help you with here?

**Comment by LubosGangster** [comment]
Hi, @TammyTee , yes you are right, thanks for the review. I have not heard of alpine js until now. I repaired it.

File: resources/views/modules/show.blade.php [hunk 1]
```diff
@@ -51,17 +51,41 @@
...
+                <div x-data="{ showMore: false }">
                     <p class="mb-3 text-xl font-semibold md:mb-7 md:text-2xl lg:text-3xl">{{ __('Skills') }}</p>
                     <ul class="flex flex-wrap -m-1 md:-m-2">
                         @forelse ($skills as $skill)
-                            <skill
-                                @if (Auth::check() && Auth::user()->hasTrack() && Auth::user()->track->modules->contains($module->id)) :completable="true" @else :completable="false" @endif
+                            @if ($loop->index < 4)
+                                <skill
+                                    @if (Auth::check() && Auth::user()->hasTrack() && Auth::user()->track->modules->contains($module->id)) :completable="true" @else :completable="false" @endif
                                 :init-completed="{{ $completedSkills->contains($skill->id) ? 'true' : 'false' }}"
```
Comment (TammyTee) [inline resources/views/modules/show.blade.php:61]:
```suggestion
                                    :init-completed="{{ $completedSkills->contains($skill->id) ? 'true' : 'false' }}"
```

File: resources/views/modules/show.blade.php [hunk 2]
```diff
@@ -51,17 +51,41 @@
...
+                                    type="{{ $skill->getMorphClass() }}"
+                                ></skill>
+                            @else
+                                <skill x-show="showMore"
+                                       @if (Auth::check() && Auth::user()->hasTrack() && Auth::user()->track->modules->contains($module->id)) :completable="true" @else :completable="false" @endif
+                                       :init-completed="{{ $completedSkills->contains($skill->id) ? 'true' : 'false' }}"
+                                       id="{{ $skill->id }}"
+                                       text="{{ $skill->name }}"
+                                       type="{{ $skill->getMorphClass() }}"
+                                ></skill>
```
Comment (TammyTee) [inline resources/views/modules/show.blade.php:73]:
```suggestion
                                <skill x-show="showMore"
                                   @if (Auth::check() && Auth::user()->hasTrack() && Auth::user()->track->modules->contains($module->id)) :completable="true" @else :completable="false" @endif
                                   :init-completed="{{ $completedSkills->contains($skill->id) ? 'true' : 'false' }}"
                                   id="{{ $skill->id }}"
                                   text="{{ $skill->name }}"
                                   type="{{ $skill->getMorphClass() }}"
                                ></skill>
```

**Review by TammyTee — APPROVED** [review]
LGTM 🎉 - I just included two small suggestions to fix code indentation but this looks and works great otherwise! Thank you!

File: app/Models/Module.php [hunk 3]
```diff
@@ -100,4 +100,5 @@ public function getNext()
             ->orderBy('id', 'asc')
             ->first();
     }
+
```
Comment (TammyTee) [inline app/Models/Module.php:103]:
```suggestion
```

**Pull request merged**